pom.xml

```diff
@@ -382,7 +382,7 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
             <configuration>
-              <additionalparam>-Xdoclint:none</additionalparam>
+              <doclint>none</doclint>
```

Comment:
I think you should now also be able to remove line 117 - `<compilerArgument>-Xlint:deprecation</compilerArgument>` for the `maven-compiler-plugin`. Can you please confirm?

Reply:
looks like it can go